src/pymortests/core/logger.py

```diff
@@ -22,5 +23,18 @@ def test_empty_log_message():
     core.logger.getLogger('test').warn('')
 
 
+def test_log_levels():
+    logger = NumpyMatrixOperator._logger
+    before_name = 'INFO'
+    logger.setLevel(before_name)
+    before = logger.level
+    with temporary_log_levels({logger.name: 'DEBUG'}):
```

Comment:
```suggestion
    with log_levels({logger.name: 'DEBUG'}):
```

Reply:
Thanks. Something's weird with how I pushed when to to this branch.